src/nibetaseries/interfaces/nilearn.py

```diff
@@ -95,3 +94,12 @@ def _run_interface(self, runtime):
         self._results['correlation_fig'] = viz_mat_path
 
         return runtime
+
+
+def _fisher_r_to_z(x):
+    import numpy as np
+    # correct any rounding errors
+    # correlations cannot be greater than 1.
+    x[x > 1.] = 1.
```

Comment:
Should this also crop negative correlation coefficients?

Reply:
Good catch, just found a numpy function that should cover the bases:
`np.clip(x, -1, 1)`

[documentation](https://docs.scipy.org/doc/numpy-1.17.0/reference/generated/numpy.clip.html)

Reply:
Awesome! I didn't know about that one. Do you mind if I make these two changes in this PR or would you prefer to do it in a separate one?